### ISSUE TITLE
Find a Doula #findadoula

### ISSUE THREAD

**Issue by jrapala** [chihacknight/breakout-groups#148]
### About the group
Find a Doula is a chihacknight creation that is helping connect Chicago-based women with a doulas to provide support during pregnancy, childbirth, and postpartum period.

### Group leaders
Dev Gosberry

### Who we're looking for
Web devs, designers, people with ideas, people who wish to learn--all are welcome!

### Tools
TBD

### Relevant Links
[https://github.com/Find-A-Doula/FindADoula](https://github.com/Find-A-Doula/FindADoula)

### Where we meet
TBD

### Chi Hack Night slack channel
You can find us on the Slack channel "findadoula"

**Comment by derekeder** [comment]
hi @jrapala - is this breakout group still active & meeting at Chi Hack Night?

**Comment by jrapala** [comment]
@derekeder Yes, we're still active and meet once or twice a month at Chi Hack Night.

**Comment by derekeder** [comment]
@jrapala great to hear!
